Recipes/JsonConfiguration/JsonConfiguration.cs

```diff
@@ -63,7 +62,7 @@ public static JsonSerializerSettings CompactSerializerSettings
                 {
                     Formatting = Formatting.None,
                     NullValueHandling = NullValueHandling.Ignore,
-                    ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                    ContractResolver = CamelCasePropertyNamesWithConstructorParametersRequiredContractResolver.Instance,
```

Comment:
Is this is a problem since we are ignoring nulls on the one hand and requiring them on the other?